attestation-service/docker/as-grpc/Dockerfile

```diff
@@ -38,6 +38,7 @@ LABEL org.opencontainers.image.source="https://github.com/confidential-container
 
 # Install Openssl Suites
 RUN apt-get update && apt-get install openssl -y && \
+    apt install --reinstall ca-certificates && \
```

Comment:
`update-ca-certificates` does not help?

Reply:
idk. I just tried `apt install --reinstall ca-certificates`

Reply:
We need to download the new `ca-certificates` deb

Reply:
I see. so this line becomes useless when the base image updates?